.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: "Go ${{ matrix.go }}"
+    strategy:
+      matrix:
+        include:
+          - go: '1.21.x'
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21.x'
```

Comment:
```suggestion
          go-version-file: 'go.mod'
```

Reply:
This is subjective. The version in `go.mod` is the minimum required Go version.

I typically argue that go.mod should aim to keep that version as _low_ as possible (to maintain compatibility), and the version used in CI should stay as _high_ as possible (to protect against regressions and vulnerabilities).

Reply:
Is License lib supposed to be used with different versions of Go?

Reply:
Not necessarily, but there's no need to artificially constrain compatibility.

Reply:
I'm with Zac here - CI should be using the most recent version of Go toolchain possible.